students/kegan/session02/series.py

```diff
@@ -0,0 +1,84 @@
+"""
+Kathryn Egan
+
+Computes Fibonacci, Lucas, and arbitrary series with
+a given first and second base case to the nth value.
+N is an ordinal starting at 1.
+"""
+
+
+def fibonacci(n):
+    """ Returns the nth value in the Fibonacci series.
+    Args:
+        n (int) : nth value to compute
+    Returns:
+        int : current value
+    """
+    if n == 1:
+        return 0
+    if n == 2:
+        return 1
+    return fibonacci(n - 2) + fibonacci(n - 1)
+
+
+def lucas(n):
+    """ Returns the nth value in the Lucas series.
+    Args:
+        n (int) : nth value to compute
+    Returns:
+        int : current value
+    """
+    if n == 1:
+        return 2
+    if n == 2:
+        return 1
+    return lucas(n - 2) + lucas(n - 1)
+
+
+def sum_series(n, base0, base1):
+    """ Returns the nth value in an arbitrary series with
+    given base cases.
+    Args:
+        n (int) : nth value to compute
+        base0 (int) : value of first in series
+        base1 (int) : value of second in series
+    Returns:
+        int : current value
+    """
+    if n == 1:
+        return base0
+    if n == 2:
+        return base1
+    return (
+        sum_series(n - 2, base0, base1) +
+        sum_series(n - 1, base0, base1))
+
+
+def test(module, solutions, **kwargs):
+    """ Tests given module against a solution.
```

Comment:
I would use the name “function “ here. “Module” has a specific ( and different) meaning in Python.